packages/odoc/odoc.1.5.0/opam

```diff
@@ -0,0 +1,53 @@
+opam-version: "2.0"
+
+version: "dev"
```

Comment:
@NathanReb I think dune-release should at least check that the version number matches. On the broader subject I still think these fields should be removed but that's another topic. I'll open a proper issue later.